modules/standard/Error.chpl

```diff
@@ -17,11 +17,32 @@
  * limitations under the License.
  */
 
+/* This module helps to handle errors. In particular, it enables routines
+   to return a syserr - encoding an error state - and then contains routines
+   that can be provided a syserr in order to print out a useful error message.
+   
+   This module defines the type syserr, which can encode an error code or an
+   error message. This type can be returned from routines generating an error.
+   
+   The IO module uses these routines in a way that supports error inspection
+   and also rapid prototyping. Most routines in the IO module have two forms.
+   In one form, an error (of type syserr) is returned in an out error argument.
+   In the second form, no error is returned, an instead the task will halt
+   with a fatal error if an error is encountered.
+   
+ */
+module Error {
+
 use SysBasic;
 
 // here's what we need from Sys
+pragma "no doc"
 extern proc sys_strerror_syserr_str(error:syserr, out err_in_strerror:err_t):c_string;
 
+/* This function takes in a string and returns it in double-quotes,
+   with internal double-quotes escaped with backslash.
+   */
+pragma "no doc"
```

Comment:
Yes, because it doesn't make any sense for it to be in the error module really, and the error module isn't trying to permanently export it. These functions use it though.